.travis.yml

```diff
@@ -25,7 +25,7 @@ env:
         - NUMPY_VERSION=stable
         - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython jinja2'
-        - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath'
+        - CONDA_ALL_DEPENDENCIES='Cython jinja2 scipy h5py matplotlib pyyaml scikit-image pandas pytz beautifulsoup4 ipython mpmath bleach'
```

Comment:
It seems bleach was not installed on Travis, so the other tests depending on it (in `test_html.py`) were not run.

Reply:
Thanks @saimn! This is a very similar issue to #6538, so could you also add a note about this optional dependency in ``docs/install.rst``?

Reply:
Preferable at a position that won't produce a conflict if any of the PRs is merged? 😄

Reply:
Good point, I have added the note, before `mock` to avoid conflicting with #6538 :wink: